begoneads.desktop

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env xdg-open
+[Desktop Entry]
+Type=Application
+Name=Begoneads
+Exec=konsole --nofork -e sudo python /home/cid0rz/programs/begoneads/begoneadsGUI.py
```

Comment:
Not everyone has konsole, nor is everyone's username cid0rz. This needs to be dynamic.

Reply:
Im working on this, The problem I am having is I am trying to move begoneadsGUI.py inside 'begoneads' subfolder, so it gets installed together with the other .py files of the package and then I am working a way of automatically detect (atleast in linux) where pip istalled begoneads, and fetch the GUI path from there. The imports get weird if i move the begoneadsGUI inside the other folder, Im trying to find a solution. Also for windows i think i can prepare a shortcut too. I havent tried the GUI on windows, I will try to test it. I have also some ideas for furhter improvements :)

Reply:
I'll await your updates :)